Configuration/PyReleaseValidation/scripts/runTheMatrix.py

```diff
@@ -165,8 +165,9 @@ def runSelected(opt):
                      29634.758,   # HLT phase-2 timing menu ticl_barrel variant
                      29634.759,   # HLT phase-2 timing menu Alpaka, single tracking iteration, LST seeding + CKF building variant
                      29634.77,    # HLT phase-2 NGT Scouting menu
-                     29634.771,   # HLT phase-2 NGT Scouting menu, with NANO:@NGTScouting
-                     29634.772]   # HLT phase-2 NGT Scouting menu, with NANO:@NGTScoutingVal
+                     29634.771,   # HLT phase-2 NGT Scouting menu, Alpaka, TICL-v5, TICL-Barrel
+                     29634.772,   # HLT phase-2 NGT Scouting menu, with NANO:@NGTScouting
+                     29634.773]   # HLT phase-2 NGT Scouting menu, with NANO:@NGTScoutingVal
```

Comment:
@bfonta , sorry can you add also `.773` here:

https://github.com/cms-sw/cmssw/blob/6a7da0e1aa9caba35aa222c5e6f3437af17be008/Configuration/PyReleaseValidation/python/relval_Run4.py#L82-L84

such that I can call it directly without the `--what upgrade`?

Reply:
Done. Please note that I changed the offsets, and the new one is ```.771``` (to keep the "NANO" ones sequentially connected).

Reply:
Thanks, noted.